pages/services/spark/2.3.1-2.2.1-2/release-notes/index.md

```diff
@@ -18,15 +18,16 @@ render: mustache
 - Updated libmesos version version with critical bug fix, [MESOS-8171](https://issues.apache.org/jira/browse/MESOS-8171)
 
 ### Documentation
-- Added a [page](https://docs.mesosphere.com/services/spark/2.3.1-2.2.1-2/limits/) documenting results from scale testing of Spark on DC/OS.
+- Added a [page](https://docs.mesosphere.com/services/spark/2.3.1-2.2.1-2/limitations/#dcos-spark-limits-test-results) documenting results from scale testing of Spark on DC/OS.
+- Added Hive integration section to [Integration page](https://docs.mesosphere.com/services/spark/2.3.1-2.2.1-2/hdfs/#adding-hive).
```

Comment:
Is this supposed to be retroactively added to this version's release notes? I think this is supposed to be part of an upcoming Spark `2.4.0-xx` release. Looking back at the jira ticket, this is supposed to include fixes for a hive Sentry bug which is something we are _currently_ working on.